app/helpers/carts_helper.rb

```diff
@@ -22,6 +22,11 @@ def display_response_actions?(cart, user)
     current_linear_approval?(cart, user)
   end
 
+  # Todo: Move this to an NCR specific template?
+  def display_restart?(cart, user)
```

Comment:
Minor: you can access `current_user` in here directly – don't need to pass it in. Unless you prefer passing it explicitly?